static/app/components/demo/demoHeader.tsx

```diff
@@ -18,12 +18,74 @@ export default function DemoHeader() {
   const sandboxData = window.SandboxData;
   // if the user came from a SaaS org, we should send them back to upgrade when they leave the sandbox
   const extraSearchParams = extraQueryParameter();
+  const walkthrough = localStorage.getItem('new-walkthrough');
 
   const collapsed = !!useLegacyStore(PreferencesStore).collapsed;
 
-  return (
-    <Wrapper collapsed={collapsed}>
-      <StyledLogoSentry />
+  let docsBtn, reqDemoBtn, signUpBtn;
+
+  if (walkthrough === 'true') {
```

Comment:
@roggenkemper Can we store `wakthrough` as `1`? IMO that's preferrable to checking the literal string `true`.

Reply:
yup! i had true as a placeholder, but will change